README.md

```diff
@@ -93,26 +93,26 @@ If desired we can stake our funds and create a validator.
 ```shell
 MONIKER="$(hostname)"
 VALIDATOR="$(palomad keys list --list-names | head -n1)"
-CHAIN_ID="paloma-testnet-5"
-DEFAULT_GAS_AMOUNT="10000000"
-VALIDATOR_STAKE_AMOUNT=100000000000ugrain
+STAKE_AMOUNT=1000000ugrain
 PUBKEY="$(palomad tendermint show-validator)"
 palomad tx staking create-validator \
-      --fees 10000000000ugrain \
+      --fees=100000ugrain \
```

Comment:
I think this can be even smaller (2000ugrain), but if I put too small a number here it will fail for some people

Reply:
Agreed.

Reply:
👍

@rootedbox when you try this can you do it with `2000ugrain`?